src/txorphanage.cpp

```diff
@@ -277,3 +277,13 @@ std::vector<std::pair<CTransactionRef, NodeId>> TxOrphanage::GetChildrenFromDiff
     }
     return children_found;
 }
+
+std::vector<TxOrphanage::OrphanTxBase> TxOrphanage::GetOrphanTransactions()
```

Comment:
My comment in the function declaration at the `txorphanage.h:84` applies here as well.

Reply:
Updated